src/braces.cpp

```diff
@@ -1103,6 +1104,13 @@ static void process_if_chain(chunk_t *br_start)
       {
          break;
       }
+
+      if (cpd.settings[UO_mod_full_brace_if_chain_only].b)
```

Comment:
Why haven't you placed it outside the loop if it depends only on the new option ?